admin/modules/user/users.php

```diff
@@ -3587,7 +3587,7 @@ function build_users_view($view)
 			}
 			if(!$user['avatar'])
 			{
-				$user['avatar'] = "../".$mybb->settings['useravatar'];
+				$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```

Comment:
It's not that important, but I created an issue: https://github.com/mybb/mybb/issues/2367